### PR TITLE
[RW-953][risk=no] Handle NPEs from invalid billing project memberships

### PR DESCRIPTION
**Pull request by rushtong**
Addresses: https://precisionmedicineinitiative.atlassian.net/browse/RW-953
Minor updates to protect against nulls in billing project memberships.

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/api/ProfileController.java [hunk 1]
```diff
@@ -268,6 +268,8 @@ private User initializeUserIfNeeded() {
     BillingProjectStatus status;
     try {
       status = fireCloudService.getBillingProjectMemberships().stream()
+          .filter(m -> m.getProjectName() != null)
+          .filter(m -> m.getCreationStatus() != null)
```
Comment (calbach) [inline api/src/main/java/org/pmiops/workbench/api/ProfileController.java]: Out of curiosity, can FC actually return null for this value? (fine to check defensively for it either way)

What line of this PR did that NPE effectively come from? While you're here, perhaps check for `fcToWorkbenchBillingMap.containsKey()` as well?

Comment (rushtong) [reply api/src/main/java/org/pmiops/workbench/api/ProfileController.java]: I don't know if FC can return a null here - this is more of a defensive attempt than root cause analysis. The associated test also triggers the error case seen in stackdriver modulo the filters.

The error is triggered from the `.findFirst()` call. If the previous map function returns a null, this stream call fails.

Comment (rushtong) [reply api/src/main/java/org/pmiops/workbench/api/ProfileController.java]: And 👍 on the `fcToWorkbenchBillingMap` idea.

**Comment by rushtong** [comment]
FYI, the codacy check is unhappy about duplicate test code. I don't agree with this particular codacy decision in test code - refactoring the boilerplate setup is an added layer of complexity that I don't think is warranted.

**Comment by calbach** [comment]
+1 to disabling that rule

**Pull request merged**